packages/documentation/.storybook/preview.js

```diff
@@ -1,5 +1,7 @@
 import { setStencilDocJson } from '@pxtrn/storybook-addon-docs-stencil';
 import docJson from '@swisspost/design-system-components/dist/docs.json';
+import postTheme from './theme';
+import { styled } from '@storybook/theming';
```

Comment:
```suggestion
import postTheme from './theme';
import { theming, styled } from '@storybook/theming';
```

We need to provide a dark and a light postTheme and override the default themes with it.
Please see: https://storybook.js.org/addons/storybook-dark-mode -> Section "Configuration"